src/store/store.ts

```diff
@@ -16,4 +17,8 @@ const store = createStore(
   applyMiddleware(thunk, promise, logger),
 );
 
+store.dispatch(setAxiosConfig({
+  baseURL: 'http://localhost:8000/api',
```

Comment:
Do you have any idea where to put the proper config to make it flexible for the development and production?

Reply:
I see two options:
1) For now we have two webpack config files for dev and prod. Maybe just put there the URL as a variable/config key until more webpack-independent settings appear on the horizon when it makes sense to create more config files?
2) use `config/app-settings.js`.

I first thought of option 1 but now I'm more into option 2. How about it?

Reply:
After considering few options I would go for 2 too. How about setting it based on `NODE_ENV` (`process.env.NODE_ENV` to be exact)?

This would require adding something like this in webpack plugins to expose it in built code:
```
new webpack.DefinePlugin({
      'process.env': ...  // 'development' or 'production' or whatever environment
    }),
```